librz/bin/format/java/class_method.c

```diff
@@ -49,10 +49,25 @@ Method *java_method_new(ConstPool **pool, ut32 poolsize, RzBuffer *buf, ut64 off
 	rz_return_val_if_fail(method, NULL);
 	method->offset = offset;
 	ut64 base = offset - rz_buf_tell(buf);
-	method->access_flags = rz_buf_read_be16(buf);
-	method->name_index = rz_buf_read_be16(buf);
-	method->descriptor_index = rz_buf_read_be16(buf);
-	method->attributes_count = rz_buf_read_be16(buf);
+	if (!rz_buf_read_be16(buf, &method->access_flags)) {
```

Comment:
Here as well - could be compressed in a macro.

Reply:
I will write a sub function.